tests/ssg_test_suite/virt.py

```diff
@@ -9,9 +9,6 @@
 
 logging.getLogger(__name__).addHandler(logging.NullHandler())
 
-# filled in by connectDomain function
-snapshots = None
-
 
 class SnapshotStack(object):
```

Comment:
didn't you singletonize the whole class? wouldn't it be better to be able to have instances of this? Having a class where all methods are classmethods smells quite bad.